client/src/pages/locations/ApprovalSteps.js

```diff
@@ -0,0 +1,52 @@
+import LinkTo from "components/LinkTo"
+import PropTypes from "prop-types"
+import React from "react"
+import { Table } from "react-bootstrap"
+
+function ApprovalSteps({ approvalSteps }) {
```

Comment:
Since this is now used in multiple places/pages, move this file to `components`.